golang/cmd/sensorconnect/iolinkDeviceFinder.go

```diff
@@ -88,7 +88,10 @@ func GetDiscoveredDeviceInformation(wg *sync.WaitGroup, i uint32) {
 	}
 	for portNumber := range portModeMap {
 		mqttRawTopic := fmt.Sprintf("ia/raw/%v/%v/X0%v", transmitterId, ddI.SerialNumber, portNumber)
-		kafkaTopic := internal.MqttTopicToKafka(mqttRawTopic)
+		validTopic, kafkaTopic := internal.MqttTopicToKafka(mqttRawTopic)
+		if !validTopic {
+			continue
```

Comment:
Can we throw a warning here?

Reply:
Resolved in https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/1045/commits/9f8ec8916a5c7c8f3ddcae9b3d9dd467c4033f12